### ISSUE TITLE
questions

### ISSUE THREAD

**Issue by wangyue663** [revanthb3000/SeizurePrediction#2]
Hello, run your code in this program, that is, the training set and test set generated using 20 main frequency similarities. Why are there so much difference between the training set and test set you provided in the code? I run the code generation The main frequency of is generally more than 1 point or more than 0 point, but the training set provided is more than 180. I don’t understand. From the frequency spectrum, the amplitude of the low frequency is obviously large. Thank you for your answer. Solve the confusion, thank you
